src/core/components/courses/TraineeTable.vue

```diff
@@ -241,9 +241,11 @@ export default {
       try {
         if (!this.firstStep) {
           this.$v.newTrainee.$touch();
-          const companyFieldError = this.addNewTraineeIdentityStep && this.$v.newTrainee.company.$error;
+          const companyOrLastnameFieldError = this.addNewTraineeIdentityStep &&
+            (this.$v.newTrainee.company.$error ||
+            this.$v.newTrainee.identity.lastname.$error);
           const newTraineeFormInvalid = !this.addNewTraineeIdentityStep && this.$v.newTrainee.$invalid;
-          if (companyFieldError || newTraineeFormInvalid) return NotifyWarning('Champ(s) invalide(s).');
+          if (companyOrLastnameFieldError || newTraineeFormInvalid) return NotifyWarning('Champ(s) invalide(s).');
```

Comment:
tu pourrais gerer ca avec `this.$v.newTrainee.$error` en mettant les bonne conditions dans ton objets `validations` qu'en penses tu ?

Reply:
done